javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount > 0 && Number.isInteger(amount)) {
+    var depositTransaction = new Transaction('deposit', amount);
+    this.transactions.push(depositTransaction);
+    return true;
+  } else return false;
+
+};
+
+Account.prototype.withdraw = function (amount) {
+  if (amount > 0 && Number.isInteger(amount)) {
+    var withdrawTransaction = new Transaction('withdrawal', amount);
+    this.transactions.push(withdrawTransaction);
+    return true;
+  } else return false;
+};
+
+Account.prototype.getBalance = function () {
+  var depositSum = 0;
+  var withdrawalSum = 0;
+
+  if (this.transactions.length === 0) {
+    var balance = 0;
+    return balance;
+  }
```

Comment:
This conditional isn't necessary. If the array's length is zero, the loop would never start, so you would still end up returning zero. You should keep the `balance` variable declaration, but remove the conditional.